.github/workflows/coverage_doc.yml

```diff
@@ -31,31 +31,27 @@ jobs:
 
       - name : Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
       - name: Build, test and generate gcov report
         run: |
+          mkdir $GITHUB_WORKSPACE/lcov_results
+          mkdir $GITHUB_WORKSPACE/lcov_results/unittest
           # add flag to include coverage
-          $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict debug build-all out-of-tree -c unit
+          sudo $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all test-add-user -c unit
```

Comment:
@mossmaurice mentioned in another context that timing_tests are not considered when calculating the test coverage. Is this correct or do we have to add them maybe here?

Reply:
Hello, if we want to activate timing test in coverage, I think we should remove the filter in tests.cmake located in folder iceoryx_meta at line 34 for module tests and line 39 for integration tests. 

I tried to add also in script run_all_tests.sh at line 135 another line with make timing_tests and it was working as well :)